web-app/js/species.show.js

```diff
@@ -299,21 +297,21 @@ function loadIndigenousData() {
                 panel.find(".profile-link").append("<a href='" + profile.url + "' title='Click to view the whole profile' target='_blank'>View the full profile</a>");
 
                 if(profile.thumbnailUrl) {
-                    panel.find(".main-image").removeClass('hidden-node');
+                    panel.find(".main-image").removeClass("hidden-node");
 
                     panel.find(".image-embedded").append("<img src='" + profile.thumbnailUrl + "' alt='" + profile.collection.title + " main image'>");
                 }
 
                 if(profile.mainVideo) {
-                    panel.find(".main-video").removeClass('hidden-node');
+                    panel.find(".main-video").removeClass("hidden-node");
```

Comment:
Nevermind those, looks like I overwrote previous changes here at some point, so had to apply them again :(